models/modules/segformer/segformer_generator.py

```diff
@@ -79,11 +79,14 @@ def __init__(
         cfg.model.train_cfg = None
         cfg.model.auxiliary_head = cfg.model.decode_head.copy()
         if self.use_final_conv:
-            num_cls = 256
+            self.num_cls_img = 256
+            self.num_cls_attn = 256
```

Comment:
Any reasons the attention be into [0,255] ? Actually it could be a float in [0,1], no ?

Reply:
Actually it's just a hard coded feature dimension, it has nothing to do with the pixel color range (in fact it would have been 3*256 for rgb values)